website/docs/typescript-support.md

```diff
@@ -17,10 +17,10 @@ Below are some guides on how to migrate an existing project to TypeScript.
 
 ## Setup {#setup}
 
-To start using TypeScript, add `@docusaurus/module-type-aliases` and some `@types` dependencies to your project:
+To start using TypeScript, add `@docusaurus/module-type-aliases` and the base TS config to your project:
 
 ```bash npm2yarn
-npm install --save-dev typescript @docusaurus/module-type-aliases @types/react @types/react-router-dom @types/react-helmet @tsconfig/docusaurus
```

Comment:
should we remove those from the TS template as well?

Are you sure it works? I don't think we have any e2e test on this yet

Reply:
`dependencies` will be installed transitively but `devDependencies` don't, so afaik it should work. I'll update the template and check it out

Reply:
Thanks, let me know if the TS template still works

Reply:
Taking longer than I thought🧐 Tried to set up a verdaccio registry locally, but when I `npm_config_registry=http://localhost:4873 yarn install`, I still get the real `2.0.0-beta.6` version on npm

Reply:
@slorber Tested, it's working🎉

Reply:
Thanks 👍 

FYI I added a readme in init package with simpler local testing instructions without Verdaccio:

---

# `@docusaurus/init`

Create Docusaurus apps easily.

## Usage

Please see the [installation documentation](https://docusaurus.io/docs/installation).

## For maintainers

For Docusaurus maintainers, templates can be tested with:

```bash
cd `git rev-parse --show-toplevel` # Back to repo root
rm -rf test-website
yarn docusaurus-init init test-website classic
cd test-website
yarn start
```

Note: `test-website` is not part of the workspace and use packages from npm.

Use the following to test the templates against local packages:

```bash
cd `git rev-parse --show-toplevel` # Back to repo root
rm -rf test-website-in-workspace
yarn docusaurus-init init test-website-in-workspace classic
cd test-website-in-workspace
yarn start
```

Reply:
Um, the problem here is whether npm installs `@types/` dependencies transitively, so I do need to mock that process of running the whole `yarn install` process

Reply:
ah I see 😅